src/Microsoft.ML.StandardTrainers/LdSvm/LdSvmTrainer.cs

```diff
@@ -0,0 +1,511 @@
+        internal const string Summary = "LD-SVM learns a binary, non-linear SVM classifier with a kernel that is specifically designed to reduce prediction time. "
+            + "LD-SVM learns decision boundaries that are locally linear.";
+
+        public sealed class Options : TrainerInputBaseWithWeight
+        {
+            /// <summary>
+            /// Depth of LDSVM Tree
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Depth of Local Deep SVM tree", ShortName = "depth", SortOrder = 50)]
+            [TGUI(SuggestedSweeps = "1,3,5,7")]
+            [TlcModule.SweepableDiscreteParam("TreeDepth", new object[] { 1, 3, 5, 7 })]
+            public int TreeDepth = Defaults.TreeDepth;
+
+            /// <summary>
+            ///  Regularizer for classifier parameter W
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Regularizer for classifier parameter W", ShortName = "lw", SortOrder = 50)]
+            [TGUI(SuggestedSweeps = "0.1,0.01,0.001")]
+            [TlcModule.SweepableDiscreteParam("LambdaW", new object[] { 0.1f, 0.01f, 0.001f })]
+            public float LambdaW = Defaults.LambdaW;
+
+            /// <summary>
+            ///  Regularizer for kernel parameter Theta
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Regularizer for kernel parameter Theta", ShortName = "lt", SortOrder = 50)]
+            [TGUI(SuggestedSweeps = "0.1,0.01,0.001")]
+            [TlcModule.SweepableDiscreteParam("LambdaTheta", new object[] { 0.1f, 0.01f, 0.001f })]
+            public float LambdaTheta = Defaults.LambdaTheta;
+
+            /// <summary>
+            ///  Regularizer for kernel parameter ThetaPrime
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Regularizer for kernel parameter Thetaprime", ShortName = "lp", SortOrder = 50)]
+            [TGUI(SuggestedSweeps = "0.1,0.01,0.001")]
+            [TlcModule.SweepableDiscreteParam("LambdaThetaprime", new object[] { 0.1f, 0.01f, 0.001f })]
+            public float LambdaThetaprime = Defaults.LambdaThetaprime;
+
+            /// <summary>
+            ///  Parameter for sigmoid sharpness
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Parameter for sigmoid sharpness", ShortName = "s", SortOrder = 50)]
+            [TGUI(SuggestedSweeps = "1.0,0.1,0.01")]
+            [TlcModule.SweepableDiscreteParam("Sigma", new object[] { 1.0f, 0.1f, 0.01f })]
+            public float Sigma = Defaults.Sigma;
+
+            /// <summary>
+            /// Indicates if we should use Bias or not in our model.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "No bias", ShortName = "noBias")]
+            [TlcModule.SweepableDiscreteParam("NoBias", null, isBool: true)]
+            public bool NoBias = Defaults.NoBias;
+
```

Comment:
There seems to be a double negative in naming here. The NoBias term defaults to false. Would it be better to name this variable UseBias and default it to true?